packages/compass-components/src/hooks/use-toast.tsx

```diff
@@ -128,19 +137,24 @@ export const ToastArea: React.FunctionComponent = ({ children }) => {
     <ToastContext.Provider value={toastActions}>
       <>{children}</>
       <>
-        {toasts.map(([id, { title, body, variant, progress }]) => (
-          <Toast
-            className={toastStyles}
-            key={id}
-            data-testid={`toast-${id}`}
-            title={title}
-            body={body}
-            variant={variant}
-            progress={progress}
-            open={true}
-            close={() => closeToast(id)}
-          />
-        ))}
+        {toasts.map(
+          ([
+            id,
+            { dataTestId, dismissible, title, body, variant, progress },
+          ]) => (
+            <Toast
+              className={toastStyles}
+              key={id}
+              data-testid={dataTestId ?? `toast-${id}`}
```

Comment:
Super nit, but I would prefer not to have this to be honest. Passed id is as good testid as any to target the element if needed and we should be checking expected content in toasts, not attributes

Reply:
Removed 👍